### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by abelemlih**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
...
+2. Confirm which environment you are deploying to e.g. test, production
+3. Verify the `plugins.yml` of the environment you are deploying to. Insure all plugins required are listed, with a correct version/branch and repository url.
+4. Verify the `config.rb` of the environment you are deploying to. Ensure all configuration customizations are included, and ensure `AppConfig[:emory_plugins_version]` is set to the version of the [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) plugin you are deploying (listed under `plugins.yml`).
+5. Create a new release of this repository and choose a tag that follows semantic versioning. You will use this new release to complete the following two tasks.
+6. Download the `config.rb` that matches the release tag above and environment you are deploying.
+7. Download the `plugins.yml` that matches the release tag and environment you are deploying.
+8. Reach out to the Lyrasis team by email, using the following template. Make sure you attach the `config.rb` and `plugins.yml` files downloaded above.
```
Comment (bwatson78) [inline README.md:38]: @abelemlih and @rotated8 Should we list the email address here, or avoid that?

Comment (rotated8) [reply README.md:38]: I think it's probably better to omit it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+    url: https://github.com/emory-libraries/aspace_static_plugins.git # url of the plugin's git repository
+```
+
+### Plugins
+
+At Emory Libraries, we manage all ArchivesSpace UI customizations under one plugin, [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins). We also use other plugins listed under `plugins.yml` of each environment e.g. [container_management_labels](https://github.com/emory-libraries/container_management_labels).
+
+### Deployment Process
+
+1. Create a new release for [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) if the latest release does not contain the most up to date changes, and ensure the new release has a tag number that follows semantic versioning. You can use the following [instructions](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository) to create a new release.
```
Comment (rotated8) [inline README.md:31]: Before this the version number needs to be updated in config.rb.

**Review by rotated8 — CHANGES_REQUESTED** [review]
Looks good, only needs version number instructions

File: README.md (hunk 2 above)
Comment (abelemlih) [reply README.md:31]: Please refer to my comment below

File: README.md [hunk 3]
```diff
@@ -1,2 +1,59 @@
...
+2. Confirm which environment you are deploying to e.g. test, production
+3. Verify the `plugins.yml` of the environment you are deploying to. Insure all plugins required are listed, with a correct version/branch and repository url.
+4. Verify the `config.rb` of the environment you are deploying to. Ensure all configuration customizations are included, and ensure `AppConfig[:emory_plugins_version]` is set to the version of the [aspace_static_plugins](https://github.com/emory-libraries/aspace_static_plugins) plugin you are deploying (listed under `plugins.yml`).
```
[the remark this answers is not in the document]
Comment (rotated8) [reply README.md:34]: @abelemlih The `AppConfig[:emory_plugins_version]` is not the version of `aspace_static_plugins`, but the version of this deployment repo. It should be the same as the release it is part of. This way, if we have changes to the deployment repo, but no changes to the plugin, we can still identify the difference two deploys on the site.

**Review by rotated8 — APPROVED** [review]
Perfect!

**Pull request merged**